src/Reports/SiteSummary.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+class SiteSummary extends SS_Report
+{
+    public function title()
+    {
+        return _t(__CLASS__ . '.TITLE', 'Installed modules');
+    }
+
+    public function description()
+    {
+        return _t(
+            __CLASS__ . '.DESCRIPTION',
+            <<<DESC
+            Provides information about what SilverStripe modules are installed,
```

Comment:
heredoc will include the whitespace here, please bring it all the way back to the left

Reply:
Whitespace should collapse in HTML output (was my logic to leaving it in), but it's not unreasonable to remove it :)